content/guides/secure-payments-by-bookingsync.md

```diff
@@ -82,7 +82,9 @@ Here's an example of the expected payload:
   "zip": "12312",
   "city": "San Francisco",
   "state": "California",
-  "country_code": "US"
+  "country_code": "US",
+  "gateway_id": "123",
```

Comment:
that will also require gateway_type: "PaymentGateway", it's a polymorphic relationship